Source/Charts/Data/Implementations/Standard/RadarChartDataEntry.swift

```diff
@@ -18,20 +18,21 @@ open class RadarChartDataEntry: ChartDataEntry
     {
         super.init()
     }
-    
+
     /// - Parameters:
     ///   - value: The value on the y-axis.
-    ///   - data: Spot for additional data this Entry represents.
-    @objc public init(value: Double, data: AnyObject?)
+    @objc public init(value: Double)
     {
-        super.init(x: 0.0, y: value, data: data)
+        super.init(x: .nan, y: value)
```

Comment:
same as pie chart. why .nan now

Reply:
Because `x` is invalid for RadarChartDataEntry and shouldn't be used. If it is being incorrectly used, this will more likely trap.